public/docs/_examples/template-syntax/dart/lib/app_component.html

```diff
@@ -487,33 +484,33 @@
 </fieldset>
 
 <div class="toe">
-    <div *ngIf="toeChoice == null">Pick a toe</div>
-    <div *ngIf="toeChoice != null">
-      You picked ...
-      <!-- #docregion NgSwitch, NgSwitch-expanded -->
-      <span [ngSwitch]="toeChoice">
+  <div *ngIf="toeChoice == null">Pick a toe</div>
+  <div *ngIf="toeChoice != null">
+    You picked ...
+    <!-- #docregion NgSwitch, NgSwitch-expanded -->
+    <span [ngSwitch]="toeChoice">
       <!-- #enddocregion NgSwitch -->
 
-        <!-- with *NgSwitch -->
-        <!-- #docregion NgSwitch -->
-        <span *ngSwitchWhen="'Eenie'">Eenie</span>
-        <span *ngSwitchWhen="'Meanie'">Meanie</span>
-        <span *ngSwitchWhen="'Miney'">Miney</span>
-        <span *ngSwitchWhen="'Moe'">Moe</span>
-        <span *ngSwitchDefault>other</span>
-        <!-- #enddocregion NgSwitch -->
-
-        <!-- with <template> -->
-        <template ngSwitchWhen="Eenie"><span>Eenie</span></template>
-        <template ngSwitchWhen="Meanie"><span>Meanie</span></template>
-        <template ngSwitchWhen="Miney"><span>Miney</span></template>
-        <template ngSwitchWhen="Moe"><span>Moe</span></template>
-        <template ngSwitchDefault><span>other</span></template>
+      <!-- with *NgSwitch -->
+      <!-- #docregion NgSwitch -->
+      <span *ngSwitchWhen="'Eenie'">Eenie</span>
+      <span *ngSwitchWhen="'Meanie'">Meanie</span>
+      <span *ngSwitchWhen="'Miney'">Miney</span>
+      <span *ngSwitchWhen="'Moe'">Moe</span>
+      <span *ngSwitchDefault>other</span>
+      <!-- #enddocregion NgSwitch -->
+
+      <!-- with <template> -->
+      <template [ngSwitchWhen]="'Eenie'"><span>Eenie</span></template>
```

Comment:
The objective was to show the result of desugaring the `*ngSwitchWhen` **without** any _optimizations_. The chapter material is complex enough, so I preferred that the **reader not be forced to recall** that if the template expression is a string constant then:
  `<foo [x]="'s'">`
is equivalent to 
  `<foo x="s">`
Besides, this is what matches the TS example. Let me know if you agree with this reasoning.